app.vue

```diff
@@ -2,19 +2,39 @@
   <v-app>
     <v-main>
       <v-container>
-        <ClientOnly>
-          <v-stage :config="configStage">
-            <v-layer>
-              <Grid :spacing="50" />
-              <Point :x="10" :y="20" />
-              <ForceVector v-for="vector in forceVectors" 
-                :tail="vector.tail" 
-                :head="vector.head" 
-              />
-              <!-- Add more points and arrows as needed -->
-            </v-layer>
-          </v-stage>
-        </ClientOnly>
+        <v-row>
+          <v-col>
+            <ClientOnly>
+              <v-stage :config="configStage">
+                <v-layer>
+                  <Grid :spacing="50" />
+                  <Point :x="10" :y="20" />
+                  <ForceVector v-for="vector in forceVectors" 
+                    :key="vector.id"
+                    :tail="vector.tail" 
+                    :head="vector.head" 
+                    :showComponents="showComponents"
+                  />  
+                </v-layer>
+              </v-stage>
+            </ClientOnly>
+          </v-col>
+          <v-col>
+            <ClientOnly>
+              <v-stage :config="configStage">
+                <v-layer>
+                  <Grid :spacing="50" />
+                  <ForceVector v-for="(vector, index) in cumulativeVectors" 
+                    :key="vector.id"
```

Comment:
Is the intention for a vector id to change? My understanding of `:key` is that it's used to basically re-render something if the value changes.

Reply:
I think if you want it to evaluate the javascript, you need the : in front of key. Otherwise, the key is set to "vector.id", the string.